sdk/src/userMap/userMap.ts

```diff
@@ -129,6 +120,12 @@ export class UserMap implements UserMapInterface {
 		return chUser.getUserAccount().authority;
 	}
 
+	public async getDLOB(slot: number): Promise<DLOB> {
```

Comment:
why does this need to be in UserMap?

Reply:
so it matched the DLOBSource interface, i could also create a separate class to do it, what do you think?